pkgs/tools/package-management/appimagekit/default.nix

```diff
@@ -90,6 +94,16 @@ in stdenv.mkDerivation rec {
     "-DBUILD_TESTING=${if doCheck then "ON" else "OFF"}"
   ];
 
+  postInstall = ''
+    cp "${squashfsTools}/bin/mksquashfs" "$out/lib/appimagekit/"
+    cp "${desktop-file-utils}/bin/desktop-file-validate" "$out/bin"
+
+    mv $out/bin/appimagetool $out/bin/appimagetool-wrapped
+    makeWrapper $out/bin/appimagetool-wrapped \
```

Comment:
Any reason not to use wrapProgram here?

Reply:
No good reason other than the fact that I wasn't aware of its existence yet ;-)